bot.py

```diff
@@ -1341,7 +1341,17 @@ async def auto_link(ctx):
 
     await ctx.respond(content=message_content, ephemeral=True)
 
-
+@bot.translate_command(
+    description = "Automatically translate any language into another using Google Translate API", aliases = ['tr']
+)
+async def translate(ctx,lang_to, *args):
+    lang_to = lang_to.lower()
+    if lang_to not in googletrans.LANGUAGES and lang_to not in googletrans.LANGCODES:
+        raise bot.BadArgument("Unfortunately the bot cannot translate this. Please contact mods and devs for more guidance.")
+    text = ' '.join(args)
+    translator = googletrans.Translator()
+    text_translator = translator.translate(text, dest=lang_to).text
+    ctx.send(text.translated)
```

Comment:
I haven't done much with the `googletrans` library, but does this not require a Google Translate API key?

Reply:
I've taken all other codes into account. Google Translate API costs money which I don't have, so unfortunately I wouldn't be able to contribute to that. I can redo the code to fit that though.

Reply:
comments*

Reply:
> I've taken all other codes into account. Google Translate API costs money which I don't have, so unfortunately I wouldn't be able to contribute to that. I can redo the code to fit that though.

Google Translate API actually has free monthly usage! Additionally, signing up gets you $300 credit.

Reply:
Oh I read it completely wrong. Ok sounds good, I'll redo the code and send it over.